### PR TITLE
Update job submit client

### PR DESCRIPTION
**Pull request by typhoonzero**
Fixes https://github.com/PaddlePaddle/cloud/issues/45

### CODE REVIEW

File: k8s/default_quota.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
```
Comment (Yancey1989) [inline k8s/default_quota.yaml]: I think we can create quota when user register automatically, but the feature can be implemented in another PR.

File: paddlecloud/paddlejob/views.py [hunk 2]
```diff
@@ -121,3 +120,63 @@ def delete(self, request, format=None):
...
+        """
+        Get logs for jobs
+        """
+        username = request.user.username
+        namespace = notebook.utils.email_escape(username)
+
+        jobname = request.query_params.get("jobname")
+        num_lines = request.query_params.get("n")
+        worker = request.query_params.get("w")
+        job_pod_list = client.CoreV1Api().list_namespaced_pod(namespace, label_selector="job-name=%s"%jobname)
```
Comment (Yancey1989) [inline paddlecloud/paddlejob/views.py]: trainer job label: https://github.com/PaddlePaddle/cloud/blob/develop/paddlecloud/paddlejob/paddle_job.py#L96  is `paddle-job: {jobname}`, chagne `label_selector="job-name=%s"%jobname ` to `label_selector="paddle-job=%s"%jobname ` ?

File: k8s/default_quota.yaml (hunk 1 above)
Comment (typhoonzero) [inline k8s/default_quota.yaml]: Sure. Quota initialization will be on next PR.

File: paddlecloud/paddlejob/views.py (hunk 2 above)
Comment (typhoonzero) [inline paddlecloud/paddlejob/views.py]: Done.
